integr8ly/application-monitoring-operator
# Modifying the Alert manager and Prometheus route name

## Description

[MGDAPI-117](https://issues.redhat.com/browse/MGDAPI-117)

Renaming of the Alertmanager and Prometheus routes to ensure they fall within the 63 character limit.

## Steps to verifiy
- Checkout this branch
- Within the Integreatly-Operator:
     - Update the `1.2.1` CSV to point to the pre-built `application-monitoring-operator` image that contains the changes in this PR: `quay.io/robrien/application-monitoring-operator:1.3.0` (line `9` and `233` of the CSV)
     - Install the RHMI operator using a `managed-api` install type, and with a namespace prefix of `redhat-managed-api`:
- Ensure that both the Alertmanager and Prometheus routes are:
     - Created without error and do not contain the word "**route**"
     - Are accessible and display the alerts are being displayed (Prometheus)
